starter-code/apple-pie-starter-code/style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  text-align: center;
+  color: white;
+  background-image: url(../images/apple-pie.jpg);
```

Comment:
Be careful: here the link is url("./images/apple-pie.jpg") and it would be the same for all the links as your css files is at the same level as your index.html.